observability/otel/otel-collector/otel-collector-exporters.adoc

```diff
@@ -91,15 +91,21 @@ The Debug Exporter prints traces and metrics to the standard output.
     exporters:
       debug:
         verbosity: detailed # <1>
+        sampling_initial: 5 # <2>
+        sampling_thereafter: 200 # <3>
+	use_internal_logger: true # <4>
     service:
       pipelines:
         traces:
-          exporters: [logging]
+          exporters: [debug]
         metrics:
-          exporters: [logging]
+          exporters: [debug]
 # ...
 ----
 <1> Verbosity of the debug export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+<2> "number of messages initially logged each second" The default value is `2` seconds.
+<3> "sampling rate after the initial messages are logged (every Mth message is logged)"
```

Comment:
```adoc
<3> The sampling rate after logging the initial messages. Logs each Nth message.
```

What's the default value?

Reply:
sampling_thereafter (default = 1): sampling rate after the initial messages are logged (every Mth message is logged). The default value of 1 means that sampling is disabled. To enable sampling, change sampling_thereafter to a value higher than 1. Refer to [Zap docs](https://godoc.org/go.uber.org/zap/zapcore#NewSampler) for more details on how sampling parameters impact number of messages.

Reply:
Added. Thank you both.